### ISSUE TITLE
Incorporate a time axis

### ISSUE THREAD

**Issue by mt-digital** [jyoshimi/history_graph#7]
Network should go from left to right in chronological order. For example
![time-axis-example](https://cloud.githubusercontent.com/assets/2425472/22567030/21d748e0-e943-11e6-9478-599ff6d38c82.png)

**Comment by jyoshimi** [comment]
Yes yes yes!    This is the big one.   Any suggestions on an implementation?   I assume the answer would come from a closer reading of these docs

http://visjs.org/docs/network/nodes.html

**Comment by mt-digital** [comment]
Great! 

I looked at the docs you posted for vis.js and there are x and y coordinates you can give to nodes (at the very bottom of the link you posted). So vis.js is definitely the first thing to try. 

d3 is also an option. It is one of those complicated-yet-powerful libraries. But it does have built-in tools for dealing with various scales, positions, etc.  vis.js should be our first choice, though, since it appears to be much simpler and you have already used it. There's probably other options I don't know about.

There are some other things to decide, like do we assign the time by their birth, years active, or something else? Birth date would probably be an easy first pass.

**Comment by jyoshimi** [comment]
I looked at d3 a few years ago and was quite impressed but never used it.  I think the first thing is to try to use what we have to get it to work, and then switch if needed.  And of course the first thing is to just get something to work roughly, then more systematicity about assignment of years can be worked out.     So basically I'm just agreeing with everything you said.
